tools/coding_standard/homepage.py

```diff
@@ -2,8 +2,13 @@
 # Utility for detecting incorrect LAMMPS homepage URLs
 #
 # Written by Richard Berger (Temple University)
-import os
+from __future__ import print_function
```

Comment:
@akohlmey not really needed if it has to run with Python 3

Reply:
that is what I thought but if I remove it and run it with python 2, I get this:
```
$python2 ../tools/coding_standard/whitespace.py
  File "../tools/coding_standard/whitespace.py", line 116
    print(line.rstrip(), file=out)
                             ^
SyntaxError: invalid syntax
```
instead of the expected:
```
$ python2 ../tools/coding_standard/whitespace.py
This script must be run with Python 3.x
```